_inc/client/components/jetpack-connect/index.jsx

```diff
@@ -18,18 +18,18 @@ const JetpackConnect = React.createClass( {
 	render: function() {
 		return (
 			<div className="jp-jetpack-connect__container">
-				<h1 className="jp-jetpack-connect__container-title" title="Please connect Jetpack to WordPress.com">
-					{ __( 'Please Connect Jetpack' ) }
+				<h1 className="jp-jetpack-connect__container-title" title="Welcome to Jetpack">
+					{ __( 'Welcome to Jetpack' ) }
 				</h1>
 
 				<Card className="jp-jetpack-connect__cta">
 					<p className="jp-jetpack-connect__description">
-						{ __( 'Please connect to or create a WordPress.com account to enable Jetpack, including powerful security, traffic, and customization services.' ) }
+						{ __( 'Please connect to or create a WordPress.com account to start using Jetpack. This will enable powerful security, traffic, and customization services.' ) }
 					</p>
 					<ConnectButton />
 					<p>
 						<a href="https://wordpress.com/start/jetpack/" className="jp-jetpack-connect__link">
-							{ __( 'No WordPress.com account? Create one for free.' ) }
+							{ __( 'No account? Create one for free...' ) }
```

Comment:
Apparently, the code is using the `…` character in general, more than the three dots. Although there are a few cases where the three dots are still present :)